apps/Gruntfile.js

```diff
@@ -32,7 +32,7 @@ var envOptions = {
   minify: (process.env.MOOC_MINIFY === '1'),
   localize: (process.env.MOOC_LOCALIZE === '1'),
   dev: (process.env.MOOC_DEV === '1'),
-  digest: (process.env.MOOC_DIGEST === '1')
+  digest: false //(process.env.MOOC_DIGEST === '1')
```

Comment:
Only that I was experimenting with this and I'm not sure it works yet.  How do I know if the Rails cachebusting is working?